dropwizard-validation/src/test/resources/logback-test.xml

```diff
@@ -0,0 +1,10 @@
+<configuration>
+    <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+        <encoder>
+            <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</pattern>
+        </encoder>
+    </appender>
+    <root level="off">
+        <appender-ref ref="STDOUT"/>
```

Comment:
I wonder, if we need to define an `<appender-ref />` here. If we turn off logging, we don't need an appender IMO. The tests work, if I remove the appender so maybe we don't need it.

Reply:
Hm, good question. This is the same `logback-test.xml` we have everywhere else. I'll raise a separate PR removing that from all of them and we can discuss the merits there.